runtime/queries/clojure/highlights.scm

```diff
@@ -1,360 +1,373 @@
- (#lua-match? @variable.builtin "^%%"))
+  (#lua-match? @variable.builtin "^%%"))
 
 ; Constructor
 ((sym_lit) @constructor
- (#lua-match? @constructor "^-%>[^>].*"))
+  (#lua-match? @constructor "^-%>[^>].*"))
 
 ; Builtin dynamic variables
 ((sym_lit) @variable.builtin
- (#any-of? @variable.builtin
-  "*agent*" "*allow-unresolved-vars*" "*assert*"
-  "*clojure-version*" "*command-line-args*"
-  "*compile-files*" "*compile-path*" "*compiler-options*"
-  "*data-readers*" "*default-data-reader-fn*"
-  "*err*" "*file*" "*flush-on-newline*" "*fn-loader*"
-  "*in*" "*math-context*" "*ns*" "*out*"
-  "*print-dup*" "*print-length*" "*print-level*"
-  "*print-meta*" "*print-namespace-maps*" "*print-readably*"
-  "*read-eval*" "*reader-resolver*"
-  "*source-path*" "*suppress-read*"
-  "*unchecked-math*" "*use-context-classloader*"
-  "*verbose-defrecords*" "*warn-on-reflection*"))
+  ; format-ignore
+  (#any-of? @variable.builtin
+    "*agent*" "*allow-unresolved-vars*" "*assert*"
+    "*clojure-version*" "*command-line-args*"
+    "*compile-files*" "*compile-path*" "*compiler-options*"
+    "*data-readers*" "*default-data-reader-fn*"
+    "*err*" "*file*" "*flush-on-newline*" "*fn-loader*"
+    "*in*" "*math-context*" "*ns*" "*out*"
+    "*print-dup*" "*print-length*" "*print-level*"
+    "*print-meta*" "*print-namespace-maps*" "*print-readably*"
+    "*read-eval*" "*reader-resolver*"
+    "*source-path*" "*suppress-read*"
+    "*unchecked-math*" "*use-context-classloader*"
+    "*verbose-defrecords*" "*warn-on-reflection*"))
 
 ; Builtin repl variables
 ((sym_lit) @variable.builtin
- (#any-of? @variable.builtin
-  "*1" "*2" "*3" "*e"))
+  (#any-of? @variable.builtin "*1" "*2" "*3" "*e"))
 
 ; Gensym
-;; Might not be needed
+; Might not be needed
 ((sym_lit) @variable
- (#lua-match? @variable "^.*#$"))
+  (#lua-match? @variable "^.*#$"))
 
 ; Types
-;; TODO: improve?
+; TODO: improve?
 ((sym_lit) @type
- (#lua-match? @type "^[%u][^/]*$"))
-;; Symbols with `.` but not `/`
+  (#lua-match? @type "^[%u][^/]*$"))
+
+; Symbols with `.` but not `/`
 ((sym_lit) @type
- (#lua-match? @type "^[^/]+[.][^/]*$"))
+  (#lua-match? @type "^[^/]+[.][^/]*$"))
 
 ; Interop
 ; (.instanceMember instance args*)
 ; (.instanceMember Classname args*)
 ((sym_lit) @function.method
- (#lua-match? @function.method "^%.[^-]"))
+  (#lua-match? @function.method "^%.[^-]"))
+
 ; (.-instanceField instance)
 ((sym_lit) @variable.member
- (#lua-match? @variable.member "^%.%-.*"))
+  (#lua-match? @variable.member "^%.%-.*"))
+
 ;  Classname/staticField
 ((sym_lit) @variable.member
- (#lua-match? @variable.member "^[%u].*/.+"))
+  (#lua-match? @variable.member "^[%u].*/.+"))
+
 ; (Classname/staticMethod args*)
 (list_lit
- .
- (sym_lit) @function.method
- (#lua-match? @function.method "^[%u].*/.+"))
-;; TODO: Special casing for the `.` macro
+  .
+  (sym_lit) @function.method
+  (#lua-match? @function.method "^[%u].*/.+"))
 
+; TODO: Special casing for the `.` macro
```

Comment:
"inline" comment moved out of the capture (not sure how to deal with this; maybe it's OK to prepare by hand (turn into inline comment)?

Reply:
I definitely moved them out of the capture before. Just need to clean the checkup for it.

Reply:
I checked the diff; it was certainly inline before this PR?

Reply:
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/29790821/f24c39af-3257-4068-9398-7a4add53a30a)
Not really, it was under a line

Reply:
Right, I must have confused this with the other moved query. Sorry about that